collect_app/src/main/res/menu/settings_menu.xml

```diff
@@ -12,13 +12,13 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
     <item
         android:id="@+id/menu_item_share"
-        android:actionProviderClass="android.widget.ShareActionProvider"
+        app:actionProviderClass="android.widget.ShareActionProvider"
```

Comment:
Why do we need changes in this file?

Reply:
This file got included in the commit as a mistake.

P.S. I'm working on resolving lint warnings and the changes in this file resolves one of the lint warnings. Using the `app:` namespace provides compatibility support for older android versions greater than minSdk.

Reply:
but it's not related to this issue. Please resolve this bug and submit a pr just with one commit since it's a small change.

Reply:
I know it's not related to the issue.

But how can undo this thing and not include this file in the commit?